src/plugins/data/public/ui/filter_bar/_global_filter_item.scss

```diff
@@ -88,3 +88,7 @@
 .globalFilterItem__popoverAnchor {
   display: block;
 }
+
+.globalFilterItem__readonlyPanel {
+  min-width: auto;
```

Comment:
This needed to fix small bug with popover.

![E1CC3E14-09F1-496E-A3DF-D6DFE2E1D6C9_4_5005_c](https://user-images.githubusercontent.com/39378793/154844678-eb38b603-cd76-4ffa-9ffa-a48f111ac93c.jpeg)